### ISSUE TITLE
[Feature proposal] Compiling a state machine from a state transition table ?

### ISSUE THREAD

**Issue by DjebbZ** [ul/vfsm#1]
Hello,

It's not a formal feature request since I don't use vsfm at all (yet), but since one can generate a state machine from a state diagram, why not from a state transition table ? The two are the visual representations of state machine. Here are some links in case you need some infos : https://en.wikipedia.org/wiki/State_transition_table / https://en.wikipedia.org/wiki/Finite-state_machine

**Comment by ul** [comment]
Yes, it is very nice to have ability to choose from alternative representations. Implemented graphml compiler is only one of possibilities, it is easy to add others. I have plans to write decent docs and examples for vfsm in the nearest future to make it easy to use & extend for everyone interested.

Speaking concretely about state transition tables, my colleague prefer using them, so I think that I will add csv/xls/whatever-popular-table-format → clj(s) compiler soon to promote using vfsm in our project.

**Comment by DjebbZ** [comment]
Happy to hear it. Will definitely make me want to use vfsm at some point in the future.
